RMAN_AL_deletion.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/sh 
+
+BASEDIR=$(dirname $0)
+PROGNAME=$(basename $0)
+DBNAME="DEVDB"
```

Comment:
We should not set the DBNAME, or ask the user to modify this file.  The DBNAME is determined dynamically by calling exec_sqlplus_get_dbname in dxh_oracle_hook.sh.  

dxh_oracle_hook.sh is the first script called; the user will actually make a hook like the following, and the DBNAME is determined from ORACLE_SID:

ORACLE_SID=DBNAME  #Done automatically for Oracle Native, but must be set for Oracle Backup Ingestion (I think?)
/home/delphix_os/dxhooks/dxh_oracle_hook.sh -o config-clone

Note: the way that sqlplus connections are managed in this project is confusing, and I think there is dead code in dxh_hook_functions.  It could use a cleanup, but that's a different RFE.